scripts/helper.py

```diff
@@ -124,4 +126,18 @@ def generate_periodic_profiles(dt_index, nodes, weekly_profile, localize=None):
 
     week_df = week_df.tz_localize(localize)
 
-    return week_df
\ No newline at end of file
+    return week_df
+
+
+def parse(l):
+    if len(l) == 1:
+        return yaml.safe_load(l[0])
+    else:
+        return {l.pop(0): parse(l)}
+
+
+def update_config_with_sector_opts(config, sector_opts):
+    for o in sector_opts.split("-"):
+        if o.startswith("CF:"):
+            l = o.split(":")[1:]
```

Comment:
We can use the snakemake function `update_config` instead, see [here](https://github.com/snakemake/snakemake/blob/9da571f29c3e4b2c77f8465edcb7e21c91f5feb7/snakemake/utils.py#L483-L504)
then we also don't need the new package `merge_deep`

something like 
`from snakemake.utils import update_config `
and
```
def update_config_with_sector_opts(config, sector_opts):
    for o in sector_opts.split("-"):
        if o.startswith("CF:"):
            l = o.split(":")[1:] 
            # TODO convert string l e.g. "sector:v2g:False" into nested dictionary {"sector":{"v2g":False}}
            update_config(config,l)
```

Reply:
Ah, neat! That's what I was looking for.